ci/matrix.yaml

```diff
@@ -8,6 +8,8 @@ workflows:
   #   - {jobs: ['test'], project: 'thrust', std: 17, ctk: 'curr', cxx: ['gcc12', 'llvm16']}
   #
   override:
+      - {jobs: ['test_lid0'], project: ['cub'],  gpu: 'h100',  sm: 'gpu', std: 'max', cxx: 'gcc12' }
```

Comment:
This overrides the current setting, so I assume this should not be merged

Reply:
Our CI status check already will block merging a PR when `override` is used :)